src/Servers/HttpSys/build.cmd

```diff
@@ -0,0 +1,3 @@
+@ECHO OFF
+SET RepoRoot=%~dp0..\..\..
+%RepoRoot%\build.cmd -projects %~dp0\**\*.*proj %*
```

Comment:
Is this file change needed, or accidental?

Reply:
Added for dev convenience, it has no impact on official builds.